video_xblock/static/html/fields/generic.html

```diff
@@ -0,0 +1,2 @@
+{# Show a textarea so we can edit it as a JSON string #}
+<textarea class="field-data-control" data-field-name="{{field.name}}" id="xb-field-edit-{{field.name}}" rows=5 cols=70>{{field.value}}</textarea>
```

Comment:
shouldwhat si the difference between this template and `text.html`? Why do we need both of them?